packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/PageScaffold/styles.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components'
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`
+
+export const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  column-gap: 32px;
+  padding: 40px 64px;
+  width: 100%;
+  max-width: 1280px;
```

Comment:
@blockdylanb  do we have a specification for the max width of a page?
or @ethan-bc should we have this specification in constellation?

Reply:
Ethan mentioned having a max width in our meeting but I don't think he had decided on an actual number at that point. Let's go with 1280 for now and adjust as needed. 👍🏻